projects/plugins/boost/app/assets/src/css/components/context-info.scss

```diff
@@ -0,0 +1,103 @@
+.jb-context-info {
+	display: inline-block;
+	position: relative;
+	color: $primary-grey;
+	font-size: 16px;
+	vertical-align: middle;
+	font-size: 1em;
+	
+
+	&__icon{
+		display: block;
+		width: 1.1em;
+		height: 1.1em;
+		font-size: calc( ( 13 / 18 ) * 1em );
+		text-align: center;
+		border-radius: 50%;
+		border: calc( ( 2 / 18 ) * 1em ) solid $gray_30;
+		cursor: pointer;
+		line-height: 1em;
+		font-weight: 700;
+		color: $gray_30;
```

Comment:
The figma file shows one instance of this in grey, and another in green. I've posted a question to @augurlu about it in the P2 thread here: https://jetpackmarketingp2.wordpress.com/2022/12/28/copy-suggestions-for-boost-to-increase-critical-css-generation-frequency/#comment-12918